docs/docs/configuration/index.md

```diff
@@ -336,9 +336,16 @@ snapshots:
       person: 15
 
 # Optional: RTMP configuration
+# NOTE: RTMP is deprecated in favor of restream
 # NOTE: Can be overridden at the camera level
 rtmp:
-  # Optional: Enable the RTMP stream (default: True)
+  # Optional: Enable the RTMP stream (default: False)
+  enabled: False
+
+# Optional: Restream configuration
+# NOTE: Can be overridden at the camera level
+restream:
```

Comment:
It was my understanding that the RTSP restream would replace RTMP for the integration restream as well, which is why it's separate. 

I can make it based on the live option if that is not in scope.

Reply:
I think I was getting mixed up because of the terminology used in the nginx file vs the config file.

The way I see it, frigate can relay or restream your video feed with multiple technologies: jsmpeg/mjpeg/rtmp/rtsp/mse/webrtc. The web UI just uses some of them.

I think we should consolidate the settings for each of them under the restream config section. The current live config will move under restream and be renamed to jsmpeg.